### PR TITLE
Create README.md File

### PR DESCRIPTION
**Pull request by kehshiba**
Task for GCI

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+* [Isaiah Rotimi Yemitan](https://wiki.openmrs.org/display/~rotimi)
+* [Daramola Ajiboye](https://wiki.openmrs.org/display/~daramola98)
+* [Korir Desmond](https://wiki.openmrs.org/display/~desmond)
+* [Titus Kipkemboi](https://wiki.openmrs.org/display/~tittoh)
+
+## Issues
+
+If you want help fix existing issues or you found a bug and want to tell us please go to
+
+https://issues.openmrs.org
```
Comment (suthagar23) [inline README.md:97]: Better to mention the direct project link here

File: README.md [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
...ogo=data%3Aimage%2Fsvg%2Bxml%3Bbase64%2CPHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHdpZHRoPSI2MTIiIGhlaWdodD0iNjEyIiB2aWV3Qm94PSIwIDAgNjEyIDYxMiI%2BPHBhdGggZD0iTTE1MyAyMjkuNWMtMjEuMTMzIDAtMzguMjUgMTcuMTE3LTM4LjI1IDM4LjI1UzEzMS44NjcgMzA2IDE1MyAzMDZjMjEuMTE0IDAgMzguMjUtMTcuMTE3IDM4LjI1LTM4LjI1UzE3NC4xMzMgMjI5LjUgMTUzIDIyOS41em0xNTMgMGMtMjEuMTMzIDAtMzguMjUgMTcuMTE3LTM4LjI1IDM4LjI1UzI4NC44NjcgMzA2IDMwNiAzMDZjMjEuMTE0IDAgMzguMjUtMTcuMTE3IDM4LjI1LTM4LjI1UzMyNy4xMzMgMjI5LjUgMzA2IDIyOS41em0xNTMgMGMtMjEuMTMzIDAtMzguMjUgMTcuMTE3LTM4LjI1IDM4LjI1UzQzNy44NjcgMzA2IDQ1OSAzMDZzMzguMjUtMTcuMTE3IDM4LjI1LTM4LjI1UzQ4MC4xMzMgMjI5LjUgNDU5IDIyOS41ek0zMDYgMEMxMzcuMDEyIDAgMCAxMTkuODc1IDAgMjY3Ljc1YzAgODQuNTE0IDQ0Ljg0OCAxNTkuNzUgMTE0Ljc1IDIwOC44MjZWNjEybDEzNC4wNDctODEuMzRjMTguNTUyIDMuMDYyIDM3LjYzOCA0Ljg0IDU3LjIwMyA0Ljg0IDE2OS4wMDggMCAzMDYtMTE5Ljg3NSAzMDYtMjY3Ljc1UzQ3NS4wMDggMCAzMDYgMHptMCA0OTcuMjVjLTIyLjMzOCAwLTQzLjkxLTIuNi02NC42NDMtNy4wMmwtOTAuMDQgNTQuMTI0IDEuMjA0LTg4LjdDODMuNSA0MTQuMTMzIDM4LjI1IDM0NS41MTMgMzguMjUgMjY3Ljc1YzAtMTI2Ljc0IDExOS44NzUtMjI5LjUgMjY3Ljc1LTIyOS41czI2Ny43NSAxMDIuNzYgMjY3Ljc1IDIyOS41UzQ1My44NzUgNDk3LjI1IDMwNiA0OTcuMjV6IiBmaWxsPSIjZmZmIi8%2BPC9zdmc%2B)](http://irc.openmrs.org)
```
Comment (suthagar23) [inline README.md:102]: Why this is so long here, Better to get rid of this part

**Review by suthagar23 — CHANGES_REQUESTED** [review]

**Comment by suthagar23** [comment]
Please follow the OpenMRS pull request tips - https://wiki.openmrs.org/display/docs/Pull+Request+Tips.

File: README.md (hunk 1 above)
Comment (kehshiba) [reply README.md]: what's the link ?

**Comment by vansha10** [comment]
@kehshiba As you have made another pull request for the same issue. You can close this one.
In the future, try to make changes to the existing PR, instead of making a new one.

**Pull request closed without merging**